keras/wrappers/scikit_learn.py

```diff
@@ -67,15 +91,26 @@ def get_params(self, deep=True):
         Parameters
         ----------
```

Comment:
It's outside of the scope of the PR, but I am reminded that this part of the code should be converted to the new style of dosctrings. This would allow us to autogenerate the documentation.